modules/core/src/v2/coins/cspr.ts

```diff
@@ -201,4 +222,63 @@ export class Cspr extends BaseCoin {
       .call(this)
       .asCallback(callback);
   }
+
+  /**
+   * Explain a Casper transaction from Raw Tx
+   *
+   * @param params
+   * @param callback
+   */
+  explainTransaction(
+    params: ExplainTransactionOptions,
+    callback?: NodeCallback<TransactionExplanation>
+  ): Bluebird<TransactionExplanation> {
+    const self = this;
+    return co<TransactionExplanation>(function*() {
+      const txHex = params.txHex || (params.halfSigned && params.halfSigned.txHex);
+      if (!txHex || !params.fee) {
+        throw new Error('missing explain tx parameters');
```

Comment:
There no an error for invalid params?

Reply:
I don't think that there is a custom error for missing or invalid parameters. Taking as an example [XTZ](https://github.com/BitGo/BitGoJS/blob/5a30f69827aa636e767250c381983769d66273a6/modules/core/src/v2/coins/xtz.ts#L267) and [XRP](https://github.com/BitGo/BitGoJS/blob/5a30f69827aa636e767250c381983769d66273a6/modules/core/src/v2/coins/xrp.ts#L288).. as you can see the error messages are set starting with lower case too.